### ISSUE TITLE
Determining Trade Size

### ISSUE THREAD

**Issue by PossibilityResult** [eljhfx/sandwich_simulator#1]
Trade size biases the fee generation so we can't just choose it arbitrarily (but could compare across different trade sizes).

Would make sense to use real Uniswap transactions but separating out between retail and arbitrage txs is not easy.

Could even make sense to abandon a simulation altogether for something more empirical.

**Comment by maxholloway** [comment]
![flow-partitions](https://user-images.githubusercontent.com/41704623/206402455-69ea8dd1-8128-4b98-8a1d-c7f97d55d574.jpeg)
source: https://twitter.com/thiccythot_/status/1597201807318867968?s=20&t=ouj9hVaH2e2DprHkxiQ29w

**Comment by maxholloway** [comment]
Yeah we could pull on-chain data; should be easy enough to dune query uniswap trade notionals. I just need to do it. 

Rn are you just running simulations sampled from something like an exponential distribution?

**Comment by maxholloway** [comment]
Separately, are you able to use your code to generate a Revenue vs. Notional Size curve? I know it's dependent on your liquidity assumptions, but that would be super dope to see.

**Comment by maxholloway** [comment]
Speaking of liquidity assumptions... I should probably also query historical Uniswap liquidity while I'm at it. Will be a challenge for V3 but doable, I remember Austin Adams had a visualization of this before and I could ask him how he generated that and see if he can share data or the scripts used

**Comment by PossibilityResult** [comment]
Revenue v. Trade size for each of these is easy (can do that).

The problem w/ empirical data is that it probably doesn't discern between retail and arb (which is hard to do).

**Comment by maxholloway** [comment]
See idea number 3 [here](https://github.com/xenophonlabs/valuing-orderflow/issues/2) for a rather eccentric Bayesian approach of identifying uninformed.

**Comment by maxholloway** [comment]
Some useful info here https://twitter.com/eljhfx/status/1583264156987576322

**Comment by maxholloway** [comment]
Another extremely simple approach is to call an order with index `i` "uninformed" if the next order in the pool is (a) within time `h` from order `i` and (b) is in the opposite direction as order `i`. 

This is dirt-simple, but it's extremely easy to compute.
